packages/react/src/stories/ic-switch.stories.mdx

```diff
@@ -44,6 +45,35 @@ import readme from "../../../web-components/src/components/ic-switch/readme.md";
   </Story>
 </Canvas>
 
+### Checked updated
```

Comment:
Please add this story to the "Web Components" storybook too

Reply:
I don't think it can be as it uses useState. I can't see examples of other components with Controlled react examples having web component equivalent stories